### PR TITLE
adding landing page files

### PR DESCRIPTION
**Pull request by slydevv**
## Description
Please include a summary of the change and relevant motivation and context. 
List any dependencies that are required for this change.

All pull requests should be related to open issues. Indicate the issue(s) bellow and remove this line:
Fixes #(issue number 1) #(issue number 2 if applicable)

## How Has This Been Tested?
Please describe in detail how you tested your changes.
Include details of your testing environment, and the tests you ran to see how your change affects other areas of the code etc.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added necessary inline code documentation
- [ ] I have added tests that prove my fix is effective and that this feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by sadiqasg — CHANGES_REQUESTED** [review]
Your strings should use double quotes.
End your blocks of code with a semi-colon;
Use strong equality sign "!=="

**Review by sadiqasg — CHANGES_REQUESTED** [review]
Dont name your folder with spaces inbetween.
Also, your ladning page should be your inital page. (i.e it should take the place of /src/App.js or be returned in it

File: client/src/landing page/App.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  color: #61dafb;
+}
+
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
```
Comment (neymarjimoh) [inline client/src/landing page/App.css:37]: Delete any pre-bulit css styles that is not needed

**Review by neymarjimoh — CHANGES_REQUESTED** [review]
Follow the codacy rules for use of double quotes instead of single quote.and semi-colons where necessary. In all, it's ok but static for now

**Pull request closed without merging**